### PR TITLE
Replace akamai cdn

### PR DESCRIPTION
**Pull request by lukaskroepfl**

### CODE REVIEW

File: player/drm/js/script.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+    'dash': 'https://cdn.bitmovin.com/content/assets/art-of-motion-dash-hls-progressive/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd',
+    'hls': 'https://cdn.bitmovin.com/content/assets/art-of-motion-dash-hls-progressive/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8',
     'smooth': 'https://test.playready.microsoft.com/smoothstreaming/SSWSS720H264/SuperSpeedway_720.ism/manifest'
   };
 
   var defaultSource = {
-    'hls': 'https://bitmovin-a.akamaihd.net/content/art-of-motion_drm/m3u8s/11331.m3u8',
-    'dash': 'https://bitmovin-a.akamaihd.net/content/art-of-motion_drm/mpds/11331.mpd',
+    'hls': 'https://cdn.bitmovin.com/content/assets/art-of-motion-dash-hls-progressive/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8',
+    'dash': 'https://cdn.bitmovin.com/content/assets/art-of-motion-dash-hls-progressive/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd',
```
Comment (jmsn) [inline player/drm/js/script.js:11]: Shouldn't we use a DRM source here too?

**Review by jmsn — CHANGES_REQUESTED** [review]

File: player/drm/demo.js [hunk 2]
```diff
@@ -6,8 +6,8 @@ var config = {
 };
 
 var source = {
-  dash: 'https://bitmovin-a.akamaihd.net/content/art-of-motion_drm/mpds/11331.mpd',
-  hls: 'https://bitmovin-a.akamaihd.net/content/art-of-motion_drm/m3u8s/11331.m3u8',
+  dash: 'https://cdn.bitmovin.com/content/assets/art-of-motion-dash-hls-progressive/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd',
+  hls: 'https://cdn.bitmovin.com/content/assets/art-of-motion-dash-hls-progressive/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8',
```
[the remark this answers is not in the document]
Comment (lukaskroepfl) [reply player/drm/demo.js:10]: very true thx for catching that, @Andr3wid do you have time to upload that? I think that one is missing on gcloud :)

File: player/drm/js/script.js (hunk 1 above)
Comment (Andr3wid) [reply player/drm/js/script.js]: [Use new URL for DRM protected AoM source](https://github.com/bitmovin/demos/pull/97/commits/fd2470ad6aef7f87e1b49b51d3b39d8957529b7b)

File: player/stream-test/js/script.js [hunk 3]
```diff
@@ -57,16 +57,16 @@ var config = {
...
   smooth: 'https://test.playready.microsoft.com/smoothstreaming/SSWSS720H264/SuperSpeedway_720.ism/manifest',
-  poster: 'https://bitmovin-a.akamaihd.net/content/MI201109210084_1/poster.jpg'
+  poster: 'https://cdn.bitmovin.com/content/assets/art-of-motion-dash-hls-progressive/poster.jpg'
 };
 
 var drmSource = {
-  dash: 'https://bitmovin-a.akamaihd.net/content/art-of-motion_drm/mpds/11331.mpd',
-  hls: 'https://bitmovin-a.akamaihd.net/content/art-of-motion_drm/m3u8s/11331.m3u8',
+  dash: 'https://cdn.bitmovin.com/content/assets/art-of-motion-dash-hls-progressive/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd',
+  hls: 'https://cdn.bitmovin.com/content/assets/art-of-motion-dash-hls-progressive/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8',
```
[the remark this answers is not in the document]
Comment (Andr3wid) [reply player/stream-test/js/script.js]: [Use new URL for DRM protected AoM source](https://github.com/bitmovin/demos/pull/97/commits/fd2470ad6aef7f87e1b49b51d3b39d8957529b7b)

File: player/drm/demo.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Andr3wid) [reply player/drm/demo.js]: Uploaded the content & replaced the DRM sources:
[Use new URL for DRM protected AoM source](https://github.com/bitmovin/demos/pull/97/commits/fd2470ad6aef7f87e1b49b51d3b39d8957529b7b)

**Review by jmsn — APPROVED** [review]
LGTM now :)

**Review by Andr3wid — APPROVED** [review]

**Pull request merged**